src/components/common/Layout/ManageLayout.tsx

```diff
@@ -13,9 +14,7 @@ export default function ManageLayout() {
   */
   // TODO: 사용자가 리로드했을 때 => 로컬 스토리지
 
-  const [isFirstTime, setIsFirstTime] = useState(true);
-
-  if (isFirstTime) return <FirstPage setIsFirstTime={setIsFirstTime} />;
+  if (loginFirst) return <FirstPage />;
```

Comment:
first가 앞에 나오는게 더 자연스러운 것 같기도 하네요!
```suggestion
  if (firstLogin) return <FirstPage />;
```